src/brain.test.js

```diff
@@ -0,0 +1,56 @@
+const { Client } = require("pg");
+
+jest.mock("pg", () => ({
+  Client: jest.fn(),
+}));
+
+const brain = require("./brain");
+
+describe("the brain", () => {
+  const connect = jest.fn();
+  const query = jest.fn();
+
+  Client.mockImplementation(() => ({ connect, query }));
+
+  beforeEach(() => {
+    connect.mockReset();
+    query.mockReset();
+  });
+
+  it("initializes", async () => {
```

Comment:
I'm not sure I understand what you mean here. Do you mean have the assertions in the `describe` instead of inside the `it`? Maybe we should sync up next week!

Reply:
yes -- happy to pair on this to make it clearer!